docs/components/geometry.md

```diff
@@ -2,25 +2,57 @@
 title: geometry
 type: components
 layout: docs
-parent_section: components
-order: 6
+order: 1
+
+parent_section: docs
+section_title: Components
+section_order: 3
 ---
 
-The geometry component provides a basic shape for an entity. The general geometry is defined by the `primitive` property. Geometric primitives, in computer graphics, means an extremely basic shape. With the primitive defined, additional properties are used to further define the geometry. A material component is usually defined alongside to provide a appearance alongside the shape to create a complete mesh.
+The geometry component provides a basic shape for an entity. The general
+geometry is defined by the `primitive` property. Geometric primitives, in
+computer graphics, means an extremely basic shape. With the primitive defined,
+additional properties are used to further define the geometry. A material
+component is usually defined alongside to provide a appearance alongside the
+shape to create a complete mesh.
```

Comment:
Double "alongside" and extra "a" in the above sentence.

How about: "A material component, to provide appearance, is usually defined alongside the shape to create a complete mesh."